### ISSUE TITLE
Version 8.13.26 causes IndexOutOfRangeException

### ISSUE THREAD

**Issue by adamhathcock** [twcclegg/libphonenumber-csharp#212]
I was updating from 8.16.20 and to dotnet 8 then starting getting this exception with my tests:
```
System.IndexOutOfRangeException
Index was outside the bounds of the array.
   at PhoneNumbers.PhoneNumberOfflineGeocoder.LoadFileNamesFromZip(Stream zipStream)
   at PhoneNumbers.PhoneNumberOfflineGeocoder..ctor(String phonePrefixDataDirectory, Assembly asm)
   at PhoneNumbers.PhoneNumberOfflineGeocoder.GetInstance()
```

Using 8.16.25 with dotnet 8 doesn't seem to cause this

**Comment by twcclegg** [comment]
Thank you for the report. I haven't resolved the issue yet but know mostly where the issue is. Should hopefully be fixed in the next release.

**Comment by adamhathcock** [comment]
Thanks!

**Comment by twcclegg** [comment]
Reverted a change in the geocoder. Should be resolved in v8.13.27
